[docs] load balancing kibana (#52659)
elastic/kibana: docs/setup/production.asciidoc

separate from  your Elasticsearch data or master nodes. To distribute Kibana
traffic across the nodes in your Elasticsearch cluster, you can run Kibana
and an Elasticsearch client node on the same machine. For more information, see

> <<load-balancing-es, Load Balancing Across Multiple Elasticsearch Nodes>>.

Reviewer: sentence case:  Load balancing across multiple Elasticsearch nodes

elasticsearch.hosts: ["http://localhost:9200"]
--------

> === Load Balancing Across Multiple Kibana Instances

Reviewer: Right now, this content appears in its own page. To include this content on the production page, include [float] above the header name.  Also, add a link to this content in the bulleted list at the top of page.

We use sentence caps for titles:  Load balancing across multiple Kibana instances

Author: 👍 ! de4f19b

> Multiple Kibana installations can be served behind a load balancer with configuration changes.  Descriptions can be referenced on the {kibana-ref}/settings.html[settings page].

Reviewer: Is there a user action in the first sentence? Would this work?

To serve multiple Kibana installations behind a load balancer, you must change the configuration.  See  {kibana-ref}/settings.html[Configuring Kibana] for details on each setting.

Author: :thu! de4f19b

Settings unique across each Kibana instances:
--------
server.uuid
server.name
--------

> Settings unique across each host (e.g. running multiple installations on the same virtual machine):

Reviewer: ```suggestion
Settings unique across each host (for example, running multiple installations on the same virtual machine):
```

--------
logging.dest
path.data
pid.file
server.port
--------

> Settings that need to be the same:

Reviewer: ```suggestion
Settings that must be the same:
```

--------
xpack.security.encryptionKey //decrypting session cookies
xpack.reporting.encryptionKey //decrypting reports stored in Elasticsearch
--------

Separate configuration files can be used from the command line by using the `-c` flag:
--------
bin/kibana -c config/instance1.yml
bin/kibana -c config/instance2.yml
--------

[float]
[[high-availability]]

> === High Availability Across Multiple Elasticsearch Nodes

Reviewer: ```suggestion
=== High availability across multiple Elasticsearch nodes
```

elasticsearch.hosts: ["http://localhost:9200"]
--------

[float]
[[load-balancing-kibana]]
=== Load balancing across multiple Kibana instances
To serve multiple Kibana installations behind a load balancer, you must change the configuration. See {kibana-ref}/settings.html[Configuring Kibana] for details on each setting.

> Settings unique across each Kibana instances:

Reviewer: instances > instance
